scripts/cat/cats.py

```diff
@@ -2435,7 +2425,7 @@ def is_potential_mate(
             ):
                 return False
 
-        age_restricted_ages = ["newborn", "kitten", "adolescent"]
+        age_restricted_ages = [CatAgeEnum.NEWBORN, CatAgeEnum.KITTEN, CatAgeEnum.ADOLESCENT]
```

Comment:
Could make this a negated helper function, `can_romance` maybe (so it'd return `not in XYZ`